Index: SWIG/_ssl.i
===================================================================
--- SWIG/_ssl.i	(revision 739)
+++ SWIG/_ssl.i	(working copy)
@@ -53,8 +53,8 @@
 extern const char *SSL_alert_desc_string_long(int);
 
 #ifndef OPENSSL_NO_SSL2
-%rename(sslv2_method) SSLv2_method;
-extern SSL_METHOD *SSLv2_method(void);
+/*%rename(sslv2_method) SSLv2_method;*/
+/*extern SSL_METHOD *SSLv2_method(void);*/
 #endif
 #ifndef OPENSSL_NO_SSL3
